doc/lua_api.txt

```diff
@@ -5832,15 +5832,88 @@ This is basically a reference to a C++ `ServerActiveObject`
 * `hud_set_hotbar_selected_image(texturename)`
     * sets image for selected item of hotbar
 * `hud_get_hotbar_selected_image`: returns texturename
-* `set_sky(bgcolor, type, {texture names}, clouds)`
-    * `bgcolor`: ColorSpec, defaults to white
-    * `type`: Available types:
-        * `"regular"`: Uses 0 textures, `bgcolor` ignored
-        * `"skybox"`: Uses 6 textures, `bgcolor` used
-        * `"plain"`: Uses 0 textures, `bgcolor` used
-    * `clouds`: Boolean for whether clouds appear in front of `"skybox"` or
-      `"plain"` custom skyboxes (default: `true`)
-* `get_sky()`: returns bgcolor, type, table of textures, clouds
+* `set_sky(parameters)`
+    * `parameters` is a table with the following optional fields:
+        * `base_color`: ColorSpec, changes fog in "skybox" and "plain".
+        * `type`: Available types:
+            * `"regular"`: Uses 0 textures, `base_color` ignored
+            * `"skybox"`: Uses 6 textures, `base_color` used as fog.
+            * `"plain"`: Uses 0 textures, `base_color` used as both fog and sky.
+        * `textures`: A table containing up to six textures in the following
+            order: top, bottom, left, right, front, back.
+        * `clouds`: Boolean for whether clouds appear in front of 
```

Comment:
Surely this applies also when `type != skybox`, or that a current limitation?

Reply:
`"regular"` gets fog from the current time of day and defined sky colors.

`"plain"` uses it for both the sky and singular sky color.

`"skybox"` only uses it for fog.

Reply:
this is a comment on the `clouds` option

Reply:
It applies to all three skybox modes.

Reply:
I see, 'in front of the skybox' implies it only works in skybox mode, this would be better as 'in front of the sky'.